docs/index.md

```diff
@@ -24,14 +24,14 @@ select
   name,
   language,
   forks_count,
-  stargazers_count
+  stargazer_count
 from
   github_my_repository;
 ```
```

Comment:
I believe this query is still incorrect as the table `github_my_repository` does not have a column `language` and the column name `forks_count` should also be in singular (`fork_count`). @bob-bot could you please make these changes as well?